common/models/User-Identity.js

```diff
@@ -145,27 +147,42 @@ export default function(UserIdent) {
         user.picture = picture;
         userChanged = true;
       }
-
       if (!githubRegex.test(provider) && profile) {
         user[provider] = getUsernameFromProvider(provider, profile);
         userChanged = true;
+        if (userChanged) {
+          return user.save(function(err) {
```

Comment:
@BerkeleyTrue is it safe to use `.save` here? I remember we [had a bug](https://github.com/FreeCodeCamp/FreeCodeCamp/pull/9111) with this before.